index.js

```diff
@@ -417,21 +423,38 @@ async function analyzeHbsFile(content, { analyzeConcatExpression = false, helper
   return translationKeys;
 }
 
-async function analyzeTranslationFiles(cwd, files) {
-  let existingTranslationKeys = new Map();
+function extractNestedPath(filepath) {
+  const regex = /translations\/(.+)\/(?:[a-z]{2}\.(?:ya?ml|json)|.*)/;
```

Comment:
Would `translations.*(yaml|yml|json)` work too?
This should match any file path with given extensions.

Reply:
regex goes way over my head so I just got this from the internet. That said, `translations\/.+\/.+(yaml|yml|json)` returns `yaml.` as the prefix

Reply:
Ah I see, I misunderstood.
This is trying to catch the middle segment :+1: